cypress-custom/support/ethereum.ts

```diff
@@ -6,7 +6,8 @@ import { Eip1193Bridge } from '@ethersproject/experimental/lib/eip1193-bridge'
 import { JsonRpcProvider } from '@ethersproject/providers'
 import { Wallet } from '@ethersproject/wallet'
 
-const TEST_PRIVATE_KEY = Cypress.env('INTEGRATION_TEST_PRIVATE_KEY')
+const TEST_PRIVATE_KEY =
+  Cypress.env('INTEGRATION_TEST_PRIVATE_KEY') || '0x34cf49001445bf8faa64a0ff3780709c2af8378e75e29c077643a387be79f089'
```

Comment:
Noooo don't add PKs to the repo 😆 

I know, for testing, etc, etc, but the sniper bots will get any funds (even if testing funds) and soon our tests will be broken again.
Better if you add that PK to your .env.local instead.

Reply:
I've essentially stolen the approach from UniSwap 😅

However fair enough, will expand on it to use a local env file.

Reply:
That's what we had until recently, but of course it was always emptied from time to time and we decided to remove it completely.
Might even be the same PK, just take a look at the file history